eras/alonzo/impl/src/Cardano/Ledger/Alonzo/Rules/Utxo.hs

```diff
@@ -347,26 +352,27 @@ utxoTransition = do
   {-   txins txb ≠ ∅   -}
   not (Set.null (getField @"inputs" txb)) ?!# InputSetEmptyUTxO
 
-  {-   feesOKp p tx utxo   -}
+  {-   feesOK pp tx utxo   -}
   feesOK pp tx utxo -- Generalizes the fee to small from earlier Era's
 
-  {-   (txins txb) ∪ (collateral txb)  ⊆ dom utxo   -}
-  eval (inputsAndCollateral ⊆ dom utxo)
-    ?! BadInputsUTxO (eval (inputsAndCollateral ➖ dom utxo))
```

Comment:
we could do something similar to before like this
(inputsAndCollateral Split.⊆ dom utxo)
    ?! BadInputsUTxO ((inputsAndCollateral Split.➖ dom utxo))

Reply:
We can't do that, because `dom utxo` is O(n) on a huge UTxO.